src/test/java/org/broadinstitute/hellbender/engine/CRAMSupportIntegrationTest.java

```diff
@@ -92,6 +93,7 @@ public void testReadCramWithIntervalsWithBAIIndex( final File cramFile, final Fi
     }
 
     private void checkReadNames( final File outputFile, final File reference, final List<String> expectedReadNames ) throws IOException {
+        SamAssertionUtils.assertValidateContentsIfCRAM(outputFile, reference);
```

Comment:
`checkReadNames()` seems like the wrong place to check the file format -- can you decouple these two checks?